plugins/inputs/interrupts/interrupts.go

```diff
@@ -50,11 +51,15 @@ func parseInterrupts(r io.Reader) ([]IRQ, error) {
 		}
 		cpucount = len(cpus)
 	}
+
 	for scanner.Scan() {
 		fields := strings.Fields(scanner.Text())
 		if !strings.HasSuffix(fields[0], ":") {
 			continue
 		}
+		if cpucount > len(fields) {
```

Comment:
I was going to say we need to compare this against `irqvals` since `fields` includes the name of the irq, but then I noticed this line from the existing tests:
```
 MIS:          0
```

So it is possible to have fewer fields than cpucount.  Maybe we just continue with no error if the ParseInt call fails?